enterprise/tailnet/pgcoord_test.go

```diff
@@ -629,10 +627,6 @@ func TestPGCoordinatorDual_Mainline(t *testing.T) {
 	err = client22.close()
 	require.NoError(t, err)
 	client22.waitForClose(ctx, t)
-
-	assertEventuallyNoAgents(ctx, t, store, agent1.id)
-	assertEventuallyNoClientsForAgent(ctx, t, store, agent1.id)
-	assertEventuallyNoClientsForAgent(ctx, t, store, agent2.id)
```

Comment:
Again, these should all be set to "LOST" in the database, not deleted, and you should assert that fact.